tree/tree/src/TTree.cxx

```diff
@@ -4839,7 +4861,7 @@ Int_t TTree::FlushBaskets() const
    Int_t nb = lb->GetEntriesFast();
 
 #ifdef R__USE_IMT
-   if (ROOT::IsImplicitMTEnabled() && fIMTEnabled) {
```

Comment:
Well I remembered wrong and Enric corrected me:

The issue here is that, in order to decide whether to create tasks in 
TTree::GetEntry, Brian only checks the flag that is local to the tree 
(fIMTEnabled). Before his change, there was an extra check of the global 
flag (ROOT::IsImplicitMTEnabled). This hierarchy of checks, which we 
discussed in the parallelisation meetings and is tested by 
ttree_read_imt, allows to disable the creation of parallel tasks only by 
setting the global flag to false.